client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -190,12 +188,33 @@ function mediaButton( editor ) {
 					return;
 				}
 
+				let useMediaWidthHeight = false;
+
+				if (
+					media.width < current.media.width ||
+					media.height < current.media.height
+				) {
+					useMediaWidthHeight = true;
+				}
+
 				// When merging, allow any updated field to be used if it doesn't
 				// already exist in the current markup, but otherwise only force
-				// update ID and URL attributes to their new values
-				const merged = assign( {}, media, current.media, pick( media, 'ID', 'URL' ), {
-					transient: !! media.transient
-				} );
+				// update ID, URL, width and height attributes to their new values
+				const merged = assign(
+					{},
+					media,
+					current.media,
+					pick(
+						media,
+						'ID',
+						'URL',
+						useMediaWidthHeight ? 'width' : '',
```

Comment:
I'd like to explicitly pass empty string `''` if `useMediaWidthHeight` is false instead of `false`.